pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -261,6 +264,13 @@ func (ctrl *PersistentVolumeController) syncClaim(ctx context.Context, claim *v1
 	}
 	claim = newClaim
 
+	if utilfeature.DefaultMutableFeatureGate.Enabled(features.VolumeAttributesClass) {
```

Comment:
Do we need add a new feature-gate to turn on/off retroactive default vac feature?

cc @sunnylovestiramisu

Reply:
I do not think it is necessary, we can support them in one feature gate. Is there any other feature that you know are using two different gates?

Reply:
@sunnylovestiramisu 

The set-default feature is implemented by the set-default admission plugin and persistentvolume controller. The admission plugin sets the default value if possible during a pvc creation and the plugin is turned off by default. The controller  sets the default value in pvc sync loop if the volumeAttributesClassName of the pvc is unset. 

if set-default feature re-use the same feature-gate, let's see how it works.

1. the admission plugin is turned off by default, and the feature-gate is turned on by users. when a default vac exists on a cluster, the volumeAttributesClassName will be set to the default value by the controller.
2. the admission plugin is turned on by user, and the feature-gate is turned on by users. when a default vac exists on a cluster, the volumeAttributesClassName will be set to the default value by the admission plugin.

In summary, whatever the admission plugin is turned on or off, the controller will set the default value if a default vac exists on a cluster when the feature-gate is turned on.

Reply:
when a default vac exists on a cluster <-- isn't this enough signal that the user wants to use this defaulting feature?